doc/ref_model/chapters/chapter04.md

```diff
@@ -4,13 +4,16 @@
 
 ## Table of Contents
 * [4.1 Compute flavours.](#4.1)
-  * [4.1.1 Storage extensions.](#4.1.1)
-* [4.2 Instance types.](#4.2)
-  * [4.2.1 B Instances (Basic).](#4.2.1)
-  * [4.2.2 N Instances (Network Intensive).](#4.2.2)
-  * [4.2.3 C Instances (Compute Intensive).](#4.2.2)
-* [4.3 One stop shop.](#4.3)
-  * [4.3.1 Naming convention.](#4.3.1)
+  * [4.1.1 Predefined Compute flavours.](#4.1.1)
+* [4.2 Network Interface Specifications.](#4.2)
+* [4.3 Storage Extensions.](#4.3)
+  * [4.3.1 Available storage extensions.](#4.3.1)
+* [4.4 Instance types.](#4.4)
+  * [4.4.1 B Instances (Basic).](#4.4.1)
+  * [4.4.2 N Instances (Network Intensive).](#4.4.2)
+  * [4.4.3 C Instances (Compute Intensive).](#4.4.3)
+* [4.5 One stop shop.](#4.5)
+  * [4.5.1 Naming convention.](#4.5.1)
 
 Infrastructure profiles are collection of capabilities, metrics, compute flavours, interface options, storage extensions, and acceleration capabilities that are offered by the infrastructure to VNFs. Infrastructure profiles are offered to VNFs in form of instance types with their corresponding options and extensions.
```

Comment:
"Infrastructure profile" is a) not defined in terminology and can easily be confused with NFVI Software Profile and NFVI Hardware Profile. Both is not intended here, I suppose. @rabiabdel please compare with your comment https://github.com/cntt-n/CNTT/pull/28#issuecomment-507197821